src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs

```diff
@@ -25,9 +25,11 @@ public static class BuildingUnit
             public const string BuildingNotFound = "GebouweenheidGebouwIdNietGekendValidatie";
             public const string BuildingStatusNotInRealized = "GebouwStatusNietInGerealiseerd";
             public const string BuildingUnitCannotBeRealized = "GebouweenheidGehistoreerdOfNietGerealiseerd";
+            public const string BuildingUnitCannotBeCorrectedFromRealized = "GebouweenheidGeplandOfNietgerealiseerdOfGehistoreerd";
```

Comment:
I think this should become "GebouweenheidNietgerealiseerdOfGehistoreerd". Planned is the desired state and we don't include it in the error code.